atomic_reactor/plugins/pre_hide_files.py

```diff
@@ -43,8 +43,28 @@ def run(self):
         self._populate_start_file_lines()
         self._populate_end_file_lines()
 
-        for parent_image_id in self.workflow.builder.parent_images.values():
-            self._update_dockerfile(parent_image_id)
+        # Find limits of each Dockerfile stage
+        self.dfp = df_parser(self.workflow.builder.df_path)
```

Comment:
I may be wrong, but it seems that `self.dfp` is initialized out of `__init__`.  Is actually dfp needed as object attribute, could be just variable?

Reply:
oh, found it. But still it should be defined in `__init__` as None at least

Reply:
(lazy evaluated property would be the best)

Reply:
Thanks, updated.
It has to be None in `__init__` because an earlier plugin modifies the file.

Reply:
I think we want to explicitly parse during `run()` to make sure we don't parse too early.

Reply:
lazy property should be parsed at first usage, that should happen somewhere inside `run()`, but I'm fine with `None`